src/fu-util-common.c

```diff
@@ -1217,6 +1217,10 @@ fu_util_device_flag_to_string(guint64 device_flag)
 		/* TRANSLATORS: we might ask the user the recovery key when next booting Windows */
 		return _("Full disk encryption secrets may be invalidated when updating");
 	}
+	if (device_flag == FWUPD_DEVICE_FLAG_NO_AUTO_REMOVE) {
+		/* TRANSLATORS: device no auto remove */
+		return _("Device no auto remove");
```

Comment:
Should the flag actually be shown to a user?  Will they be able to do anything with the information?
If so, the translators comment and the comment a user sees need to be clarified.

Reply:
Am not sure the purpose of this flag and if this should be informed to user , please let me know what would be the appropriate message and translator here